### PR TITLE
Some fix for windows

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl [hunk 1]
```diff
@@ -857,6 +862,8 @@ $(NPM_CMD):
 	$(PRERULE_CMD)
 ifeq ($(OPERATING_SYSTEM), LINUX)
 	npm install npm || sleep 20 || npm install npm || sleep 90 || npm install npm
+else
+	touch $@
```
Comment (kalbermattenm) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: Shouldn't we document the fact that Windows users should create a `npm` file in the root folder of their project ? Or is this file automatically created ?

Comment (sbrunner) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: The file is automatically created :-)

Comment (kalbermattenm) [inline c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl]: Nice ! Thanks ;-) (I am discovering `touch`...)

**Comment by fredj** [comment]
LGTM

**Pull request merged**